reconcile/terraform_users.py

```
@@ -110,6 +115,28 @@ def _filter_participating_aws_accounts(
+            policy_name = user_policy.get("name", "unknown")
+            policy_document = user_policy.get("policy")
+            if policy_document is not None and (
+                isinstance(policy_document, dict)
+                or (isinstance(policy_document, str) and policy_document.strip())
+            ):
+                try:
+                    validate_aws_policy(policy_document, f"user_policy-{policy_name}")
+                except AWSPolicyValidationError as e:
+                    # Re-raise with additional context while preserving original errors
+                    context_message = f"Policy validation failed in terraform_users integration for role '{role.get('name', 'unknown')}' user policy '{policy_name}'"
+                    enhanced_errors = [context_message] + list(e.errors)
```

Ruff (0.15.15)

[warning] Consider `[context_message, *list(e.errors)]` instead of concatenation

Replace with `[context_message, *list(e.errors)]`

(RUF005)